cmd/snapd-generator/main.c

```diff
@@ -89,8 +89,6 @@ static int ensure_root_fs_shared(const char *normal_dir)
 	fprintf(f, "Where=" SNAP_MOUNT_DIR "\n");
 	fprintf(f, "Type=none\n");
 	fprintf(f, "Options=bind,shared\n");
-	fprintf(f, "[Install]\n");
```

Comment:
Maybe a small comment here like: `// implicitly run when /snap/** gets mounteed, no need for a [Install] section` or something along these lines?

Reply:
Well, `[Install]` will never do anything no matter what. It is not supposed to be in generated units. If we want to install to a target, we have to create the symlink. I can however add a comment that explains why we do not create symlinks.

Reply:
I have added a comment.